package.json

```diff
@@ -7,6 +7,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
+    "start": "start-storybook -p 6006 -s .storybook/assets",
```

Comment:
2 lines with the same content can be bad for maintenance, we should either remove the storybook line, or just make yarn start be an alias by using `"start": "yarn storybook"`